fracatal/functional_jax.py

```diff
@@ -55,6 +55,43 @@ def ft_convolve(grid, kernel, default_dtype=np.float32):
                                         
   return convolved 
 
+def compute_entropy(subimage):
```

Comment:
Issue #17 
Of these metrics, frequency entropy seems to capture details most relevant to interesting features.